README.rst

```diff
@@ -10,48 +10,95 @@ pytest-picked
     :target: https://ci.appveyor.com/project/anapaulagomes/pytest-picked/branch/master
     :alt: See Build Status on AppVeyor
 
-Run the tests related to the changed files (according with Git)
+.. image:: https://img.shields.io/pypi/v/pytest-picked.svg
+    :target: https://pypi.org/project/pytest-picked/
+    :alt: See Package Status on PyPI
 
-----
+Run the tests related to the modified files (according to Git)
 
-This `pytest`_ plugin was generated with `Cookiecutter`_ along with `@hackebrot`_'s `cookiecutter-pytest-plugin`_ template.
+---
 
+.. image:: demo.gif
+    :target: https://asciinema.org/a/cWV3sx6poG4aaPZfyVI8nOJ7J
+    :height: 400px
+    :alt: Demo
 
-Features
---------
+Let's say you have the following output from your ``git status``:
```

Comment:
Maybe remove the `your`:
`Let's say you have the following output from ``git status``:`